README.md

```diff
@@ -11,49 +11,163 @@
 
-  * [Installing the client library](#installing-the-client-library)
+#### `config`
+*Config Object* <br>
+**Required.**
 
+**Config Object**
+|      field      |     type  	|   description	|
+|---------------	|-----------	|-------------	|
+|   upstreamRepo	|   `string`	| **Required.** The repository to suggest changes to.  |
+|   upstreamOwner	|   `string`	| **Required.** The owner of the upstream repository. |
+|   description	  |   `string`	| The GitHub Pull Request description. Default is `'code suggestions'`.  |
+|   title       	|   `string`	| The GitHub Pull Request title. Default is `'chore: code suggestions'`.      |
+|   branch	      |   `string`	| The branch containing the changes. Default is `'code-suggestions'`.   |
+|   message     	|   `string`	| The commit message for the changes. Default is `'code suggestions'`. |
+|   force	        |   `string`	| Whether or not to force push the reference even if the ancestor commits differs. Default is `false`. |
 
-* [Versioning](#versioning)
-* [Contributing](#contributing)
-* [License](#license)
 
-## Quickstart
 
-### Installing the client library
+#### `logger`
+*[Logger](https://www.npmjs.com/package/@types/pino)* <br>
+The default logger is [Pino](https://github.com/pinojs/pino). You can plug in any logger that conforms to [Pino's interface](https://www.npmjs.com/package/@types/pino)
+
+
+## CLI
+
+### Installation
 
 ```bash
-npm install @google-cloud/code-suggester
+npm i code-suggester -g
 ```
 
+### code-suggester pr
+`code-suggester pr` - opens a GitHub Pull Request containing the a set of files.
+
+#### Syntax
+
+
+`code-suggester pr [options] --upstream-repo=<string> --upstream-owner=<string>`
+
+
+#### Options
 
+#### `--upstream-repo, -r`
+*string* <br>
+**Required.** The repository to create the fork off of.
 
 
+#### `--upstream-owner, -o`
+*string* <br>
+**Required.** The owner of the upstream repository.
+
+
+#### `--description, -d`
+*string* <br>
+The GitHub Pull Request description. *Default* value is: `code suggestions`.
+
+#### `--title, -t`
+*string* <br>
+The GitHub Pull Request title. *Default* value is: `chore: code suggestions`.
+
+#### `--branch, -b`
+*string* <br>
+The GitHub working branch name. *Default* value is: `code-suggestions`.
+
+#### `--message, -m`
+*string* <br>
+The GitHub commit message. *Default* value is: `code suggestions`.
+
+#### `--force, -f`
+*boolean* <br>
+Whether or not to force push a reference with different commit history before the remote reference HEAD. *Default* value is: `false`.
+
+#### `--files [<file1>...]`
+*string* <br>
+**Required.** A list of files
+
+**Note:** Use either `--files` or `--git-dir` exclusively. Using both with terminate with an error.
+
+#### `--git-dir`
+*string* <br>
+**Required.** The path of a git directory
+
+**Note:** Use either `--files` or `--git-dir` exclusively. Using both with terminate with an error.
+
+
+### Example
+```
+code-suggester pr --o='Foo' --r='Bar' --git-dir="/my-project"
```

Comment:
Usually a single character option uses a single dash and doesn't use an equals sign:
```suggestion
code-suggester pr -o 'Foo' -r 'Bar' --git-dir="/my-project"
```